vals.go

```diff
@@ -221,7 +223,9 @@ func (r *Runtime) Eval(template map[string]interface{}) (map[string]interface{},
 			{
 				path2 := uri.Path
 				path2 = strings.TrimPrefix(path2, "#")
-				path2 = strings.TrimPrefix(path2, "/")
+				if host != "" {
```

Comment:
Ah! So you treat `host != ""` as the indicator for the `ref+tfstate://relative/path` pattern, assuming `ref+tfstate:///absolute/path` would result in `host == ""`, right?

Reply:
Exactly!